packages/contracts/package.json

```diff
@@ -15,7 +15,7 @@
         "test": "run-s compile build run_mocha",
         "run_mocha": "mocha 'lib/test/**/*.js' --timeout 10000 --bail --exit",
         "compile:comment": "Yarn workspaces do not link binaries correctly so we need to reference them directly https://github.com/yarnpkg/yarn/issues/3846",
-        "compile": "node ../deployer/lib/src/cli.js compile --contracts-dir src/contracts --artifacts-dir src/artifacts",
+        "compile": "node ../deployer/lib/src/cli.js compile --contracts Exchange,DummyToken,ZRXToken,Token,WETH9,TokenTransferProxy,MultiSigWallet,MultiSigWalletWithTimeLock,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,MaliciousToken,TokenRegistry --contracts-dir src/contracts --artifacts-dir src/artifacts",
```

Comment:
Let's use npm config as we do for artifacts list in 0x.js https://github.com/0xProject/0x.js/blob/development/packages/0x.js/package.json#L34
It'll be easier to read and edit later